plugin/k8s_external/setup.go

```diff
@@ -3,6 +3,8 @@ package external
 import (
 	"strconv"
 
+	prs "github.com/coredns/coredns/plugin/pkg/parse"
+
```

Comment:
has extra newline and why are you rewriting the import?

Reply:
you mean : why I name the import "prs" ?
=> there is a confustion with the function parse .. but i can rename that one. ok.